src/OMODFrameworkWrapper.cpp

```diff
@@ -393,7 +394,7 @@ OMODFrameworkWrapper::EInstallResult OMODFrameworkWrapper::install(MOBase::Guess
           }
         }
 
-        scriptData->Pretty(%omod, dataPath, pluginsPath);
+        scriptData->Pretty(% omod, dataPath, pluginsPath);
```

Comment:
I think your formatting tool didn't know that it was formatting C++/CLI instead of C++. Like the `*` operator when we dereference a pointer, we don't want a space between `%` and its operand.

Reply:
I disabled formatting for this repository because of C++/CLI, I must have messed up my commit, I'll fix.